Project EFT/Database/DBConnector.cs

```diff
@@ -33,6 +33,18 @@ public static void Init()
             if (!GetNumberUsersInDB()) throw new Exception("Could not get initial number of users in DB");
         }
 
+        public static void InitForTests()
+        {
+            string[] lines = System.IO.File.ReadAllLines("../../../../../info.txt");
+            connectionString = @"server=" + lines[0] + ";userid=" + lines[1] + ";password=" + lines[2] + ";database=" + lines[3];
+
+            // cache list of all problems currently in db
+            if (!GetProblemsList()) throw new Exception("Could not read initial problem list from DB");
+
+            // cache number of users in db
+            if (!GetNumberUsersInDB()) throw new Exception("Could not get initial number of users in DB");
+        }
+
```

Comment:
not sure if the gitignore solution would fix this entirely - I can see the case where the actual init no longer needs "../../" before info.txt, but the test init still needs the first three, though that's just a guess